akka-sample-javaslang/src/main/java/sample/javaslang/Master.java

```diff
@@ -0,0 +1,54 @@
+package sample.javaslang;
+
+import akka.actor.Props;
+import akka.actor.UntypedAbstractActor;
+import javaslang.collection.List;
+import javaslang.collection.Stream;
+
+import static sample.javaslang.Minion_StoppedPatterns.*;
+
+import static javaslang.API.*;
+import static javaslang.Predicates.*;
+
+public class Master extends UntypedAbstractActor {
+
+  private final Maze maze;
+
+  private List<Minion.Stopped> results = List.empty();
+
+  public Master(Maze maze) {
+    this.maze = maze;
+  }
+
+  public void onReceive(Object message) {
```

Comment:
is it possible to implement this in terms of `AbstractPartialFunction` so that it can be used with the ordinary `AbstractActor`? See prototype by @jypma here https://github.com/Tradeshift/ts-reaktive/pull/50

Note that `AbstractActor.Receive` is just a wrapper around a `PartialFunction`.